entity-framework/core/modeling/types-and-tables.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Types and Tables - EF Core
+author: rowanmiller
+ms.date: 10/27/2016
+ms.assetid: cbe6935e-2679-4b77-8914-a8d772240cf1
+uid: core/modeling/types-and-tables
+---
+# Types and Tables
+
+Including a type on your context means that it is included in EF's model; we usually refer to such a type as an *entity*. EF can read and write entity instances from/to the database, and if you're using a relational database, EF can create tables for your entities via migrations.
+
+## Including types in the model
+
+By convention, types that are exposed in DbSet properties on your context are included in your model as entities. Types that are specified in the `OnModelCreating` method are also included, as are any types that are found by recursively exploring the navigation properties of other discovered types.
+
+In the code sample below, all types are included:
+
+* `Blog` is included because it's exposed in a DbSet property on the context.
+* `Post` is included because it's discovered via the `Blog.Posts` navigation property.
+* `AuditEntry` because it is specified in `OnModelCreating`.
+
+<!-- [!code-csharp[Main](samples/core/Modeling/Conventions/Samples/IncludedTypes.cs?highlight=3,7,16)] -->
+``` csharp
+class MyContext : DbContext
+{
+    public DbSet<Blog> Blogs { get; set; }
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        modelBuilder.Entity<AuditEntry>();
+    }
+}
+
+public class Blog
+{
+    public int BlogId { get; set; }
+    public string Url { get; set; }
+
+    public List<Post> Posts { get; set; }
+}
+
+public class Post
+{
+    public int PostId { get; set; }
+    public string Title { get; set; }
+    public string Content { get; set; }
+
+    public Blog Blog { get; set; }
+}
+
+public class AuditEntry
+{
+    public int AuditEntryId { get; set; }
+    public string Username { get; set; }
+    public string Action { get; set; }
+}
+```
+
+## Excluding types from the model
+
+If you don't want a type to be included in the model, you can exclude it:
+
+# [Data Annotations](#tab/data-annotations)
+
+<!-- [!code-csharp[Main](../../../samples/core/Modeling/DataAnnotations/Samples/IgnoreType.cs?highlight=20)] -->
+``` csharp
+[NotMapped]
+public class BlogMetadata
+{
+    public DateTime LoadedFromDatabase { get; set; }
+}
+```
+
+# [Fluent API](#tab/fluent-api)
+
+<!-- [!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/Samples/IgnoreType.cs?highlight=12)] -->
+``` csharp
+protected override void OnModelCreating(ModelBuilder modelBuilder)
+{
+    modelBuilder.Ignore<BlogMetadata>();
+}
+```
+
+***
+
+> [!NOTE]
```

Comment:
Looking at this again, I'm not even sure we need a note here saying the rest is relational.

Reply:
How are we planning to integrate cosmos here for container mapping? I don't have preference for title/note/something else. Though we should have something so that users can differentiate Containers  for cosmos too.

Reply:
I think having Cosmos configuration separate still makes sense since it's relevant to a much smaller audience

Reply:
Agree that Cosmos stuff belongs in a Cosmos doc page - even SQL Server-specific stuff (e.g. in-memory tables) belongs in provider-specific. We could consider linking to those pages, though I'm not sure it's necessary.

Am removing the "this is relational" comment for now.